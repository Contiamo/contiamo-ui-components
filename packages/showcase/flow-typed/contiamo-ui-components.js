declare module "contiamo-ui-components" {
  declare var exports: any
}
