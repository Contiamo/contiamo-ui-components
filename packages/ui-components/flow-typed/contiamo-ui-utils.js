declare module "contiamo-ui-utils" {
  declare var exports: any
}
