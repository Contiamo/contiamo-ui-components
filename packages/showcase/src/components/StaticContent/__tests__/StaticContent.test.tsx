import * as React from "react"
import { render } from "enzyme"

import { wrapTheme } from "contiamo-ui-utils"
import { contiamoTheme } from "contiamo-ui-components"
import ThemelessStaticContent from "../StaticContent"

const StaticContent = wrapTheme(contiamoTheme)(ThemelessStaticContent)

describe("App Showcase: StaticContent", () => {
  it("Should render correctly", () => {
    expect(render(<StaticContent markdownContent="## Hello" />)).toMatchSnapshot()
  })
})
