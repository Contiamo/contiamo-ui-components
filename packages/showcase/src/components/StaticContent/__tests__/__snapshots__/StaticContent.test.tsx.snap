// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Showcase: StaticContent Should render correctly 1`] = `
<div
  class="css-13van8m"
>
  <h2
    id="hello"
  >
    Hello
  </h2>
  

</div>
`;
